example_test.go

```diff
@@ -1230,3 +1233,26 @@ func ExampleConnection_CloseGraceful_force() {
 	// Result:
 	// <nil> connection closed by client (0x4001)
 }
+
+func ExampleConnection_watchOnce() {
```

Comment:
It will help to assign the example to the request type in godoc:
```suggestion
func ExampleWatchOnceRequest() {
```

Reply:
Updated.